contrib/plotinus/template.py

```diff
@@ -0,0 +1,17 @@
+pkgname = "plotinus"
+pkgver = "0.2.0"
+pkgrel = 0
+build_style = "cmake"
+# Vendored dependency
+cmake_dir = "unityx/plotinus"
+hostmakedepends = ["cmake", "ninja", "pkgconf", "vala"]
+makedepends = ["gtk+3-devel"]
+pkgdesc = "Searchable command palette in every modern GTK+ application"
+maintainer = "Earldridge Jazzed Pineda <earldridgejazzedpineda@gmail.com>"
+license = "GPL-3.0-or-later"
+url = "https://github.com/p-e-w/plotinus"
+source = "https://gitlab.com/ubuntu-unity/unity-x/unityx/-/archive/main/unityx-main.tar.gz?path=unityx/plotinus>plotinus.tgz"
```

Comment:
you can't package the 'main' branch of things and then pretend it's '0.2.0'; you would at least need a git ref and then to make the version something like 0.2.0_git<somedate>

the `>plotinus.tgz` rename is also wrong, since on any upgrade it would still have the old tarball with the same name cached

Reply:
There aren't any version numbers in the source code, all I can do is take the original author's word for it.